dashdotdb/services/deploymentvalidation.py

```diff
@@ -34,18 +33,18 @@ def insert(self, token, validation=None):
     def _insert(self, token, item):
         if 'metric' not in item:
             self.log.error('skipping validation: key "metric" not found')
-            return
```

Comment:
Make the error message match the log message, so:

return 'key "metric" not found', 400

Reply:
Also, add 400 to the return codes in the swagger.yaml